db/migrate/20201010084621_remove_support_webmentions_from_articles.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSupportWebmentionsFromArticles < ActiveRecord::Migration[6.0]
+  def change
+    safety_assured { remove_column :articles, :support_webmentions, :boolean }
```

Comment:
Rather than adding a migration for removing this column, lets just remove all the migrations from the PR if we are not going to be doing anything with the database. That will help keep the codebase nice and clean.